core/trino-main/src/main/java/io/trino/operator/scalar/MathFunctions.java

```diff
@@ -1408,6 +1408,30 @@ public static Double cosineSimilarity(
         return dotProduct / (normLeftMap * normRightMap);
     }
 
+    @Description("Calculates the cosine distance between the give sparse vectors")
+    @ScalarFunction
+    @SqlNullable
+    @SqlType(StandardTypes.DOUBLE)
+    public static Double cosineDistance(
+                    @OperatorDependency(
```

Comment:
The formatting is off here.  Take a look at the cosineSimilarity method

Reply:
Fixed